docs/src/content/docs/guides/migrating_using_cli_and_kubectl.md

```diff
@@ -0,0 +1,421 @@
+
+
+### Gather Information
+
+For creating the resources, we will need some values to put into the YAML manifests. Let’s gather those values first.
+
+
+#### Gather the credentials to VMware vCenter and OpenStack
+
+Check the resource name for the VMware vCenter credentials using the following command:
+
+``` 
+kubectl get vmwarecreds -n <namespace>
+```
+
+**Note**: The default namespace for vJailbreak ecosystem is `migration-system`.
+
+Copy the name of the `VMwareCreds` resource that refers to the vCenter hosting the VM you want to migrate. This will be used when creating the MigrationTemplate. For example, if following is the output of the above command:
+
+```
+$ kubectl get vmwarecreds -n migration-system
+NAME       STATUS
+vCenter-a  Succeeded
+```
+
+Then, the name of the `VMwareCreds` would be `vCenter-a`.
+
+Similarly, check the resource name for the OpenStack credentials using the following command:
+
+```
+kubectl get openstackcreds -n <namespace>
+```
+
+
+Copy the name of the `OpenstackCreds` resource that refers to the OpenStack where you want to migrate the VM. This will also be used when creating the MigrationTemplate.
+
+For example, if following is the output of the above command:
+
+```
+$ kubectl get openstackcreds -n migration-system
+NAME            STATUS
+OpenStack-a     Succeeded
+```
+
+Then the name of the `OpenStackcreds` would be `OpenStack-a`.
+
+
+#### Gather the details of the VM to be migrated
+
+Get the details of the VM using `VMwareMachine` resource and details are correct:
+```
+kubectl get vmwaremachine -n <namespace> <vm-name> -o yaml
+```
+
+From the YAML, note down the following:
+* VM's name using `spec.vms.name`
+* Get the datastores under `spec.vms.datastores`. This will be used for storage mapping.
+* Get the networks under `spec.vms.networks`. This will be used for network mapping.
+* (Optional) Get the Operating System type under `spec.vms.osType`. The reason for keeping it optional is that vJailbreak can auto-detect the OS most of the time, if the VM is turned on in VMware. For cold migration or migration of a VM that is in powered off state, this field will be required.
+
+**Note**: Make sure that the `spec.vms.vmState` is set to `running`, otherwise the migration will not take place.
+
+For example, if following is the output of the above command: 
+
+```
+$ kubectl get vmwaremachines -n migration-system vm-1 -o yaml
+
+apiVersion: vjailbreak.k8s.pf9.io/v1alpha1
+kind: VmwareMachine
+metadata:
+  name: vm-1
+  namespace: migration-system
+  creationTimestamp: "2025-05-27T09:27:34Z"
+  generation: 1
+  labels:
+    pcd-vm-OpenStack-a: d3f3d5c7-f6d8-410e-bb9f-a7afe4j8nhg
+    vmwarecreds.k8s.pf9.io-vCenter-a: "true"
+spec:
+  vms:
+    cpu: 1
+    memory: 4096
+    name: vm-1
+    osType: linuxGuest
```

Comment:
```suggestion
    osFamily: linuxGuest
```

Reply:
Thanks @OmkarDeshpande7 . It has been updated.